TackleTabby/Assets/Scripts/UI/Encyclopedia/Encyclopedia.cs

```diff
@@ -82,27 +89,67 @@ private void OnFishButtonPressed(EncyclopediaFishButton fishButton)
         foreach (EncyclopediaFishButton button in FishButtons)
             button.Exit();
 
-        bool hasCaught = _fishProgress.TryGetValue(fishButton.FishType, out CaughtFish caught);
-
         DescriptionText.enabled = false;
-        OpenFishInfo();
         
-        FishNameDisplay.SetText(fishButton.FishType.DisplayName);
-        FishDisplayImage.sprite = fishButton.FishType.FishSprite;
-        if (hasCaught)
-            FishSizeDisplay.SetText($"{caught.CaughtSize:F1} inch");
+        if (_fishProgress.TryGetValue(fishButton.FishType, out CaughtFish caught))
+        {
+            DisplayAlreadyCaughtFish(caught);
+        }
+        else
+        {
+            DisplayLockedFish(fishButton.FishType);
+        }
```

Comment:
these don't need the brackets
```suggestion
        if (_fishProgress.TryGetValue(fishButton.FishType, out CaughtFish caught))
            DisplayAlreadyCaughtFish(caught);
        else
            DisplayLockedFish(fishButton.FishType);
```

Reply:
Indeed true, I just think it reads better and more clear when an "if" is followed by an "else"  to use the brackets explicitly